README.md

```diff
@@ -79,6 +79,15 @@ Alternatively, for the SIGMORPHON 2016 shared task data format:
 
 this format is specified by `--features-col 2 --features-sep , --target-col 3`.
 
+## Model checkpointing
+
+Checkpointing is largely handled by [Lightning](https://pytorch-lightning.readthedocs.io/en/stable/common/checkpointing_basic.html). The path for model information, including checkpoints, is specified by a combination of 
+`--model_dir` and `--experiment`, such that we build the path `model_dir/experiment_name/version_*n*`, where each run of an experiment with the same model_dir and experiment_name is namespaced with a new version number. Within a version, we store everything needed to reload a model, namely the hyperparameters (`model_dir/experiment_name/version_*n*/harapams.yaml`), and a directory of checkpoints (`model_dir/experiment_name/version_*n*/checkpoints`).
```

Comment:
typo: "harapams".

Is your intention for the n to be bolded or italiced here? Unfortunately that does not seem to work in GitHub's markdown formulation.

Reply:
Yeah, oops I should have double checked when I rendered it. Will remove.